test/suites/smoke-test-dancelight/test-external-validator-slashes.ts

```diff
@@ -0,0 +1,91 @@
+import "@tanssi/api-augment/dancelight";
+
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import type { ApiPromise } from "@polkadot/api";
+import { PRIMARY_GOVERNANCE_CHANNEL_ID, getCurrentEraStartBlock, getPastEraStartBlock } from "utils";
+
+describeSuite({
+    id: "SMOK09",
+    title: "Smoke tests for external validators slashes pallet",
+    foundationMethods: "read_only",
+    testCases: ({ it, context, log }) => {
+        let api: ApiPromise;
+
+        beforeAll(async () => {
+            api = context.polkadotJs();
+        });
+
+        it({
+            id: "C01",
+            title: "Number of bonded eras does not exceed bonding duration",
+            test: async () => {
+                const bondedEras = await api.query.externalValidatorSlashes.bondedEras();
+                const bondedDuration = await api.consts.externalValidatorSlashes.bondingDuration;
```

Comment:
I don't think we need await here, because `consts` is not a Promise.

Reply:
will check